Formula/faas-cli.rb

```diff
@@ -35,8 +35,8 @@ def install
     system "go", "build", *std_go_args(ldflags: ldflags), "-a", "-installsuffix", "cgo"
     bin.install_symlink "faas-cli" => "faas"
 
-    (bash_completion/"faas-cli").write Utils.safe_popen_read(bin/"faas-cli", "completion", "--shell", "bash")
-    (zsh_completion/"_faas-cli").write Utils.safe_popen_read(bin/"faas-cli", "completion", "--shell", "zsh")
+    generate_completions_from_executable(bin/"faas-cli", "completion",
+                                         shells: [:bash, :zsh], shell_parameter_format: :arg)
```

Comment:
Again, I think it's better to just pass `--shell` as an arg like this instead of relying on knowing the `shell_parameter_format` rules.

```suggestion
    generate_completions_from_executable(bin/"faas-cli", "completion", "--shell", shells: [:bash, :zsh])
```